src/en/controllers-config.md

```diff
@@ -68,8 +78,91 @@ mongo-memory-profile         | string | low      | low/default              | Se
 set-numa-control-policy      | bool   | false    | false/true               | Sets whether numactl is preferred for running processes with a specific NUMA (Non-Uniform Memory Architecture) scheduling or memory placement policy for multiprocessor systems where memory is divided into multiple memory nodes
 state-port                   | integer | 37017   |                          | The port to use for mongo connections
 
+### Excluding information from the audit log
+
+See [Audit logging][troubleshooting-logs-audit] for background information on
+this topic.
+
+Excluding information from the audit log is done via the
+`audit-log-exclude-methods` key above, which refers to API calls/methods. The
+recommended approach for configuring the filter is to view the log and make a
+list of those calls deemed undesirable. There is no definitive API call list
+available in this documentation.
+
+The default value of key `audit-log-exclude-methods` is the special value of
+'ReadOnlyMethods'. As the name suggests, this represents all read-only events.
+
+An example value for this key is: `[Pinger.Ping]` where a log message intended
```

Comment:
This explanation seems completely back to front. We just told people they should examine the current log to see wht things to exclude, so start with the example log message, then explain how to remove it

Reply:
:ok: